### ISSUE TITLE
Remove value options?

### ISSUE THREAD

**Issue by jongacnik** [jongacnik/gr8#10]
Perhaps it is more clear if we actually remove all *value* options and require overwriting utilities using the `add` method instead:

```js
gr8.add({
  prop: 'font-size',
  vals: [3, 2, 1]
})
```

I feel like this would clarify what the `add` method does, as well as prevent confusion why only *some* things are editable via options.

**Comment by jondashkyle** [comment]
I’m into this. Since the `add` method can be used to overwrite utilities, and would now be used to modify defaults, perhaps it could be renamed to something more generic, such as `utility`.

```
gr8.utility({
  prop: 'font-size',
  vals: [3, 2, 1]
})
```

**Comment by jongacnik** [comment]
resolved #17 

ended up keeping value options since things like margin and padding are quite complex utils and having to redefine all that yourself seems counter
